app/views/articles/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1><%= @article.title %></h1>
+
+<p><%= @article.body %></p>
+
+<ul>
+  <li><%= link_to "Edit", edit_article_path(@article) %></li>
+  <li>
+    <%= link_to "Destroy", article_path(@article), method: :delete, data: { confirm: "Are you sure?" } %>
+  </li>
+</ul>
+
+<h2>Comments</h2>
+<% @article.comments.each do |comment| %>
+  <%= render @article.comments %>
+<% end %>
+
+<h2>Add a comment:</h2>
+<%= render 'comments/form' %>
```

Comment:
Remember single quotes and the empty line at the end of the file.

Reply:
done